automap/src/comm_layer/pcp_pmp_common/mod.rs

```diff
@@ -393,13 +396,25 @@ pub mod tests {
         let result = subject.execute_command("dir booga");
 
         match result {
-            Err(stderr)
+            Err(Either::Left(stderr))
                 if stderr.contains("The system cannot find the file specified")
                     || stderr.contains("No such file or directory") =>
             {
                 ()
             }
-            Err(stderr) => panic!("Unexpected content in stderr: '{}'", stderr),
+            x => panic!("Expected error message in stderr; got {:?}", x),
+        }
+    }
+
+    #[test]
+    fn find_routers_command_works_when_error_is_returned() {
```

Comment:
You think so? `ErrorKind::NotFound` is just a regular Rust standard-library error, not an OS error.

If I've misunderstood you, let me know; otherwise, I'll leave it like this.